miasm2/analysis/machine.py

```diff
@@ -51,10 +51,10 @@ def __init__(self, machine_name):
             mn = arch.mn_armt
             from miasm2.arch.arm.ira import ir_a_armtb as ira
         elif machine_name == "sh4":
-            from miasm2.arch.sh4.disasm import dis_sha4 as dis_engine
+            dis_engine = None
```

Comment:
This since unnecessary, since `dis_engine` has already been set to `None`. Same for `ira` at line 57.